### PR TITLE
[MIG] project_task_materials_analytic_partner module: migration to 10.0

### PR DESCRIPTION
**Pull request by cubells**
Depends on
- [x] #288 
- [x] OCA/account-analytic#85

Migration of module project_task_materials_analytic_partner to version 10.0

- Change module name
- Update README
- Update views

cc @Tecnativa

### CODE REVIEW

File: project_task_material_analytic_partner/README.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+This module adds the partner on task materials and propagate it to the
+auto-generated analytic entries on the "Other partner" field.
+
+Usage
+=====
+
+Go to Project > Tasks, and you will find the partner field in material lines.
+
+Using this partner, when you consume these materials, the target of the invoice
```
Comment (pedrobaeza) [inline project_task_material_analytic_partner/README.rst]: Change this text to reflect better the purpose of the module (as in 10.0, there's no invoicing of analytic lines):
```
Using this partner, when you consume the materials, the other partner field in the generated analytic lines will be this one instead the one in the project.
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by cubells** [comment]
All is :green_heart: 

Ready to be approved.  :smile: 

cc @Tecnativa

**Comment by pedrobaeza** [comment]
Runbot is still complaining about a missing module. I'm rebuilding again to see.

**Comment by pedrobaeza** [comment]
No luck. Please be sure you have properly rebase and if so, close this PR and open another one.

**Comment by cubells** [comment]
Close this pr in order to rebuild runbot correctly with a new pr: https://github.com/OCA/project/pull/310

**Pull request closed without merging**